index.html

```diff
@@ -7,12 +7,8 @@
     <link rel="icon" type="images/x-icon" href="./public/img/helmet.png" />
     <link rel="stylesheet" href="public/style.css" />
   </head>
-
   <body>
     <div class="quiz-container" id="quiz"></div>
-    <div class="rocket-background">
-      <img src="public/img/rocket.png" alt="" />
-    </div>
```

Comment:
why did you removed the rocket `dive`? was it by mistake?

Reply:
i didn't delete actually i have no idea :( can you add while merging if it is possible?